File: src/main/java/io/github/tanguygab/playerlistexpansion/filters/Whitelisted.java
package io.github.tanguygab.playerlistexpansion.filters;

import org.bukkit.OfflinePlayer;

public class Whitelisted extends Filter {

    @Override
    public boolean filter(String name, OfflinePlayer viewer) {
        OfflinePlayer player = getOffline(name);
        return player != null && player.isWhitelisted();
    }
}

File: src/main/java/io/github/tanguygab/playerlistexpansion/filters/Whitelisted.kt
package io.github.tanguygab.playerlistexpansion.filters

import org.bukkit.OfflinePlayer

class Whitelisted : Filter() {
    override fun filter(name: String?, viewer: OfflinePlayer?): Boolean {
        val player = getOffline(name)
        return player != null && player.isWhitelisted
    }
}
